.goreleaser.jsonnet

```diff
@@ -0,0 +1,63 @@
+local tools = ['tempo', 'tempo-query', 'tempo-cli'];
```

Comment:
i'd prefer leaving this yaml. it's easier to read.

Reply:
Templating. Lots of repetition with all three tools. And it would get even worse with OS packaging. I don’t mind moving it back to YAML if maintaining a big YAML file isn’t an issue

Reply:
I prefer the simplicity of the yaml file to the additional toolchain dependencies of jsonnet and having this file in repo. perhaps when i see the full file i will change my mind? I didn't find the repetition of the previous config to be that terrible.

Reply:
I removed the jsonnet part